pages/linux/mcookie.md

```diff
@@ -0,0 +1,19 @@
+# mcookie
+
+> Generate a 128-bit random hexadecimal number.
+
+- Generate random number:
+
+`mcookie`
+
+- Generate random number, use destination file as a seed for the randomness:
+
+`mcookie -f {{path/to/file1}} -f {{path/to/file2}}`
+
+- Generate random number, read specific number of bytes from destination file and use it as a seed for the randomness:
```

Comment:
Perhaps this will read better?
```suggestion
- Generate a random number, using a specific number of bytes from a file as a seed for the randomness:
```